app/com/m3/octoparts/repository/ConfigsRepository.scala

```diff
@@ -25,14 +25,34 @@ trait ConfigsRepository {
   def findParamById(id: Long): Future[Option[PartParam]]
 
   /**
-   * Find a single ThreadPoolConfig by id
+   * Find a single ThreadPoolConfig by id.
+   *
+   * All retrieved items are populated with their [[HystrixConfig]]
    */
-  def findThreadPoolConfigById(id: Long): Future[Option[ThreadPoolConfig]]
+  def findThreadPoolConfigById(id: Long): Future[Option[ThreadPoolConfig]] = findAllConfigs().map {
```

Comment:
Other than the config listing page, I don't believe we are doing full retrieves of all the configs (which includes related tables in this case !), though I could be wrong.

Is it possible to look for HystrixConfigs with a given threadPoolId instead ? I believe ThreadPoolConfig is loaded automatically when going through that route (e.g. loading a HystrixConfig loads a ThreadPoolConfig), so maybe we can reconstruct a ThreadPoolConfig by "in reverse" (grouping on HystrixConfig#threadPoolId).

Should we drop the ORM already ?